WordPressKit/WordPressComRestApi.swift

```diff
@@ -421,6 +422,100 @@ open class WordPressComRestApi: NSObject {
         let fileURL = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent(fileName)
         return fileURL
     }
+
+    // MARK: - Async
+
+    private lazy var urlSession: URLSession! = {
```

Comment:
The implicitly unwrapped optional type definition shouldn't be necessary (compiles on my end). Is there a particular reason why we need it?

```suggestion
    private lazy var urlSession: URLSession = {
```